### ISSUE TITLE
pretrained yolo model

### ISSUE THREAD

**Issue by ruinianxu** [jianing-sun/Mask-YOLO#1]
Hi Jianing,
I got your network working but trained from scratch since I can't load weight file of either just mobilenet backend or whole yolo with mobilenet. I wonder whether you can provide the pretrained model you used for building maskyolo. 
Thank you so much for your help.
Best

**Comment by jianing-sun** [comment]
@ruinianxu Hi, sorry for the delay, just saw your message. Unfortunately I don't have the pretrained model on hand as this is an old project, but there is a tip for fast training: train the yolo branch first, and that would be super fast to finish. Then train the whole maskyolo model, loaded with the pretrained yolo model. I remember there is a parameter controlled which part of the neural nets you want to train, something like `==yolo` / `==training` / `==inference`.

Hope it helps. 
Jianing
